README.md

```diff
@@ -64,9 +49,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Microverse 
 
 ## 📝 License
```

Comment:
it's great you included the MIT License as guided 🥳.  
unfortunately, the link to the Licence is broken.

- [ ] Kindly add the MIT.MD file to your project to fix the broken link.

Reply:
I already have the MIT.md file in my project

Reply:
@niicode please check the comment again.